### ISSUE TITLE
Issue with bans gr:miner command

### ISSUE THREAD

**Issue by UserDC-LeGrand** [MaynardMiner/SWARM#402]
When issuesin the following command: "bans remove gr:xmrig-cpu", it looks like the bans command does not recognize gr as a valid algorithm, which it is. It assumes it's a coin instead....

![image](https://user-images.githubusercontent.com/99490445/196847965-9398971e-3f07-4ab6-9ea6-8de5f83bda97.png)

**Comment by MaynardMiner** [comment]
interesting. You can try 'ghostrider' instead. I will make it so ban is accepted regardless of name, though. This is a bug in that it should be able to accept stat screen naming schemas.

**Comment by MaynardMiner** [comment]
Can you open up pool-algos.json in configs\pools folder, find ``gr`` in the .json, and see what its "exclusions" are listed as?

I'm not sure if this is a bug for the ``get stats`` command and how it outputs bans, or if its a bug with the ``bans`` command. It depends on how the ban was implemented. It's been so long since I worked on this, I don't remember- Seeing "exclusions" for ``gr`` in pool-algos.json will help greatly.

**Comment by MaynardMiner** [comment]
nvm. I figured it out. The ban is not an exclusion- It was a timeout ban that was added to .\timeout. It should be removed when you run this command, but it's not.

There are different ways SWARM does bans. If it's user-derived (argument or command), it adds it under "exclusions". This way it is permanent until user removes.

If SWARM implements the ban, the details is added under .\timeout so it can self-manage the ban. The issue here is that ``bans`` doesn't do anything with that folder, but it should.
